sympy/stats/crv_types.py

```diff
@@ -1974,13 +1977,30 @@ def Normal(name, mean, std):
     >>> simplify(std(2*X + 1))
     2
 
+    >>> m = Normal('X', [1, 2], [[2, 1], [1, 2]])
+    >>> from sympy.stats.joint_rv import marginal_distribution
+    >>> pprint(density(m)(y, z))
+              2          2
+             y    y*z   z
+           - -- + --- - -- + z - 1
+      ___    3     3    3
+    \/ 3 *e
+    ------------------------------
+                 6*pi
+
+    >>> marginal_distribution(m, m[0])(1)
+     1/(2*sqrt(pi))
+
+
     References
     ==========
 
     .. [1] http://en.wikipedia.org/wiki/Normal_distribution
     .. [2] http://mathworld.wolfram.com/NormalDistributionFunction.html
     """
 
+    if isinstance(mean, (list)) and isinstance(std, list):
```

Comment:
needs to check against matrices as well, I believe there's a common class, `MatrixBase` I think it's the name.